packages/vx-demo/src/docs-v2/examples/vx-gradient/Example.tsx

```diff
@@ -20,11 +21,17 @@ const defaultMargin = {
   bottom: 0,
 };
 
-const numColumns = 5;
-
-const gradientIds = [
-  [...new Array(numColumns)].map((_, idx) => `vx-gradient-demo-row-0-column-${idx}`),
-  [...new Array(numColumns)].map((_, idx) => `vx-gradient-demo-row-1-column-${idx}`),
+const Gradients: React.FC<{ id: string }>[] = [
```

Comment:
in cleaning up the `Pattern` example I used a similar pattern (hah) for the `Gradient` demo which is cleaner